device-types/Dell/Unity-XT-380.yaml

```diff
@@ -0,0 +1,36 @@
+---
+manufacturer: Dell
+model: Unity XT 380
+slug: dell-unity-xt-380
+part_number: 100-566-075-03
+u_height: 2
+is_full_depth: true
+subdevice_role: parent
```

Comment:
To make sure we have the most complete information, can you add the following parameters, `airflow` `weight` and `weight_unit`